granulate_utils/metadata/databricks_client.py

```diff
@@ -11,21 +11,23 @@
 
 import requests
 
+from granulate_utils.exceptions import DatabricksJobNameDiscoverException
+from granulate_utils.metadata import Metadata
+
 HOST_KEY_NAME = "*.sink.ganglia.host"
-DATABRICKS_DEPLOY_CONF_PATH = "/databricks/common/conf/deploy.conf"
 DATABRICKS_METRICS_PROP_PATH = "/databricks/spark/conf/metrics.properties"
 CLUSTER_TAGS_KEY = "spark.databricks.clusterUsageTags.clusterAllTags"
-JOB_NAME_KEY = "RunName"
```

Comment:
Use this const again?

Reply:
b86cfe3

Reply:
unresolved
please re-review this a78ddb3
@michelhe @Jongy

Reply:
oops wrong location